### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by jakebailey**
This is going to be a moderate overhaul / cleanup of dependencies to try and make things more maintainable and movable over to dt-tools.

### CODE REVIEW

**Review by rondales — APPROVED** [review]

File: src/graphql-client.ts [hunk 1]
```diff
@@ -14,7 +13,7 @@ const headers = {
 };
 
 const cache = new InMemoryCache();
-const link = new HttpLink({ uri, headers, fetch });
+const link = new HttpLink({ uri, headers, fetch: fetch as unknown as typeof globalThis.fetch });
```
[the remark this answers is not in the document]
Comment (jakebailey) [reply src/graphql-client.ts:16]: I uh don't remember why I did this

[the remark this answers is not in the document]
Comment (jakebailey) [reply src/graphql-client.ts]: Removed 😅

File: tsconfig.json [hunk 2]
```diff
@@ -3,8 +3,8 @@
         "outDir": "dist",
         "strict": true,
         "module": "commonjs",
```
Comment (jakebailey) [inline tsconfig.json:5]: This is not good, but our imports of apollo break thanks to their broken types: https://arethetypeswrong.github.io/?p=%40apollo%2Fclient%403.9.11

**Comment by jakebailey** [comment]
Hm, I'm not 100% sure that pnpm's `node_modules` layout is going to work with Azure Functions...

File: tsconfig.json (hunk 2 above)
Comment (andrewbranch) [reply tsconfig.json:5]: Odd, since they _use_ @arethetypeswrong/cli

**Comment by jakebailey** [comment]
I reverted the pnpm change since I'm wanting to upgrade things before redeploying. pnpm will have to be a later change (if at all) due to the symlink problem and Azure Functions only doing zipfiles.

**Pull request merged**